src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -170,13 +205,40 @@ fn test_build_user_pb_buf() -> anyhow::Result<()> {
         println!("{:?}", buf);
     }
 
+    // Stage on GCS, supported in version >=4.
+    {
+        let gcs_stage_info = test_user_stage_info_gcs();
+        let p = gcs_stage_info.to_pb()?;
+        let mut buf = vec![];
+        common_protos::prost::Message::encode(&p, &mut buf)?;
+        println!("{:?}", buf);
+    }
+
     Ok(())
 }
 
 #[test]
 fn test_load_old_user() -> anyhow::Result<()> {
     // built with `test_build_user_pb_buf()`
 
+    {
+        // User information generated by test_build_user_pb_buf()
+        let user_info_v4: Vec<u8> = vec![
+            10, 9, 116, 101, 115, 116, 95, 117, 115, 101, 114, 18, 9, 108, 111, 99, 97, 108, 104,
+            111, 115, 116, 26, 25, 18, 17, 10, 13, 116, 101, 115, 116, 95, 112, 97, 115, 115, 119,
+            111, 114, 100, 16, 1, 160, 6, 4, 168, 6, 1, 34, 26, 10, 18, 10, 8, 10, 0, 160, 6, 4,
+            168, 6, 1, 16, 2, 160, 6, 4, 168, 6, 1, 160, 6, 4, 168, 6, 1, 42, 15, 8, 10, 16, 128,
+            80, 24, 128, 160, 1, 160, 6, 4, 168, 6, 1, 50, 15, 8, 1, 18, 5, 114, 111, 108, 101, 49,
+            160, 6, 4, 168, 6, 1, 160, 6, 4, 168, 6, 1,
+        ];
+        let p: pb::UserInfo =
+            common_protos::prost::Message::decode(user_info_v4.as_slice()).map_err(print_err)?;
+        let got = mt::UserInfo::from_pb(p).map_err(print_err)?;
+        let want = test_user_info();
+        println!("{:?}", got);
+        assert_eq!(want, got);
+    }
+
```

Comment:
Is UserInfo changed in this PR?

Remove the println.

Reply:
It's not changed.